### ISSUE TITLE
The loss of the network

### ISSUE THREAD

**Issue by wanjinchang** [natanielruiz/deep-head-pose#35]
dear author:
   I am confused of the loss back pass of the network. Is it the three separate losses  back pass alternately??? Hope for you reply...

**Comment by natanielruiz** [comment]
There are three losses for the network, one for each angle (since there are 3 fully-connected layers). Each fully-connected layer is updated using it's own loss, and the convolutional backbone is updated with all 3 losses. Is this clear? You can check the paper for more info.

Cheers

**Comment by natanielruiz** [comment]
Yes, the 3 losses one by one at each backprop iteration.

**Comment by wanjinchang** [comment]
@natanielruiz I try to reproduce your paper using TensorFlow, the following is my loss during training,but when I test on AFLW2000, the mae is over 10, expecially the yaw mae,is about 18.Can you please tell me your idea???
![loss](https://user-images.githubusercontent.com/12625246/45790438-6edfd500-bcb7-11e8-97e9-5a02b26c9586.png)
